runatlantis.io/docs/api-endpoints.md

```diff
@@ -0,0 +1,151 @@
+# API Endpoints
+
+Aside from interacting via pull request comments, Atlantis could respond to a limited number of API endpoints.
+The API endpoints are disabled by default, since the API endpoint could change the infrastructure directly.
+To enable API endpoints, `api-secret` should be configured.
+
+:::tip Prerequisites
+
+* You have set `api-secret` as part of the [Server Configuration](server-configuration.html#api-secret)
+* You have pass `X-Atlantis-Token` with the same secret in the request header
+  :::
+
+## Endpoints
+
+### POST /api/plan
+
+#### Description
+
+Execute [atlantis plan](using-atlantis.html#atlantis-plan) on the specified repository.
+
+#### Parameters
```

Comment:
Couldn't we implement swagger/openapi docs on the Atlantic server so all endpoints could be hittable and self documenting?

Reply:
I would love to see this! Maybe we can have a ticket to track it separately.